packages/audits/src/components/AuditsRawTable.tsx

```diff
@@ -42,13 +42,28 @@ export const AuditsRawTable: FC<IAuditsRawTable> = React.memo(({ headerProps, ..
       );
   }, [headerProps]);
 
+  const getTableData = useCallback(() => {
+    const columnsName = headerProps.map(({ name }) => name);
+    let tableData = [...data];
+
+    for (const columnName of columnsName) {
+      tableData = tableData.map((item) => ({
+        ...item,
+        [columnName]: typeof item[columnName] === 'object' ? JSON.stringify(item[columnName]) : item[columnName],
```

Comment:
I think you should verify its not null as well.  In case it will be null you will change the data to "null"